src/live.asp.net/gulpfile.js

```diff
@@ -43,3 +45,15 @@ gulp.task("min:css", function () {
 });
 
 gulp.task("min", ["min:js", "min:css"]);
+
+gulp.task('hint', function() {
+    gulp.src(paths.js)
+        .pipe(jshint())
+        .pipe(jshint.reporter())
+});
+
+gulp.task('lint', function() {
```

Comment:
You should make these new tasks run as part of the "prepublish" step in `project.json`.